plaid/errors.go

```diff
@@ -4,17 +4,20 @@ import (
 	"fmt"
 )
 
-type plaidError struct {
+type PlaidError struct {
+	APIResponse
+
 	// List of all errors: https://github.com/plaid/support/blob/master/errors.md
-	ErrorCode int    `json:"code"`
-	Message   string `json:"message"`
-	Resolve   string `json:"resolve"`
+	ErrorType      string `json:"error_type"`
```

Comment:
could you clarify?

Reply:
I think what we have here is good. I was just saying that (even though its a little weird to have `Error.ErrorType`, its probably better to have it match the API jsons response schema exactly.

It would be more idiomatic to have it be `Error.Type`, but it is probably better to be true to the API here.

Reply:
Gotcha, sounds good